.goreleaser.yml

```diff
@@ -31,10 +31,71 @@ archives:
     format_overrides:
       - goos: windows
         format: zip
+snapcrafts:
+  - name: parca
+    # Switch to true once snap ownership is sorted
+    publish: false
+    summary: Parca continuous profiling tool
+    description: |
+      Continuous profiling for analysis of CPU and memory usage, down to the line
+      number and throughout time. Saving infrastructure cost, improving performance,
+      and increasing reliability.
+    grade: stable
+    confinement: strict
+    license: Apache-2.0
+    base: core22
+    # Channels in store where snap will be pushed.
+    # Default depends on grade:
+    # * `stable` = ["edge", "beta", "candidate", "stable"]
+    # * `devel` = ["edge", "beta"]
+    channel_templates:
+      - edge
+      - "{{ if not .IsSnapshot }}stable{{ end }}"
```

Comment:
This is now the only thing I'm not sure about - I don't know what'll happen if this tries to publish. Still spelunking in goreleaser.